### ISSUE TITLE
Connection error on Unpublish

### ISSUE THREAD

**Issue by marcin-adamczewski** [pedroSG94/rtmp-rtsp-stream-client-java#851]
Hi @pedroSG94 !
I'm getting the `onConnectionFailedRtmp()` callback when I receive the `NetStream.Unpublish.Success` message from the server. I noticed that it's called in the RtmpConnection class when such a message is received.
https://github.com/pedroSG94/rtmp-rtsp-stream-client-java/blob/master/rtmp/src/main/java/com/github/faucamp/simplertmp/io/RtmpConnection.java#L743

I'm wondering why it is implemented like this?
Basically, it causes some issues in my app. The server sends this message when a stream disconnects. As I'm calling `retry` when getting `onConnectionFailedRtmp` it tries to retry after disconnected. It also messes up a little bit with analytics.

What do you think about providing some other callback for this message? It seems like this message doesn't mean that there was a connection failure.

**Comment by pedroSG94** [comment]
According with this:
https://helpx.adobe.com/adobe-media-server/ssaslr/netstream-class.html
That method should be ignored instead of declarate a callback or anything. I will fix it

**Comment by pedroSG94** [comment]
Fixed:
https://github.com/pedroSG94/rtmp-rtsp-stream-client-java/commit/62d24d8ee550218978669815d3422aa0e7615f74

**Comment by marcin-adamczewski** [comment]
Cool, thanks!
